src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
> How can I add in some space between the text and the icon with the above code?

@HirdayGupta I am sure `letter-spacing` will resolve this.

Example: `letter-spacing: 2px;`